application/tests/unit/common/models/UserTest.php

```diff
@@ -305,5 +305,35 @@ public function testPasswordChangeLog()
         $this->assertEquals(PasswordChangeLog::SCENARIO_RESET, $log->scenario);
     }
 
+    public function testIsEmailInUseByOtherUser()
+    {
+        $user1 = $this->users('user1');
+        $user2 = $this->users('user2');
+
+        $this->assertFalse($user1->isEmailInUseByOtherUser($user1->email));
+        $this->assertTrue($user1->isEmailInUseByOtherUser($user2->email));
+        $this->assertFalse($user1->isEmailInUseByOtherUser('made-up-email@domain.com'));
+
+    }
+
+    public function testRefreshPersonnelDataForUserWithSpecificEmail()
+    {
+        /*
+         * Test user who does not have updated information in personnel
+         */
+        $user1 = $this->users('user1');
+        User::refreshPersonnelDataForUserWithSpecificEmail($user1->email);
+        $notupdated = User::findOne(['id' => $user1->id]);
```

Comment:
I don't quite understand what's going on here and in the lines below, but presumably that's because I don't know the fixture details for these users. Is there a way to make this test more clear? (It may not be worth the effort.)